.python-version

```diff
@@ -1 +1 @@
-3.8.1
+3.8.2
```

Comment:
Sneaky, different PR me thinks

Reply:
I'll leave this till we discuss the approach at the next tech session